NEXT_CHANGELOG.md

```diff
@@ -104,7 +104,24 @@ By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/p
 
 ## 🛠 Maintenance
 
-### Update apollo-router-scaffold to use the published router crate [PR #1782](https://github.com/apollographql/router/pull/1782)
+### Improve error message when querying non existent field [Issue #1816](https://github.com/apollographql/router/issues/1816)
+
+When querying a non existent field you will get a better error.
+
+```patch
+{
+  "errors": [
+    {
+-       "message": "invalid type error, expected another type than 'Named type Computer'"
++       "message": "cannot query field 'xxx' on type 'Computer'"
```

Comment:
I think this more accurately reflects the new message?

```suggestion
+       "message": "Cannot query field \"xxx\" on type \"Computer\""
```

Reply:
I think so. That's JSON isn't it?